mdn/kumascript.py

```diff
@@ -522,21 +523,44 @@ class CSSBox(KnownKumaScript):
     expected_scopes = set()
 
 
-class CSSxRef(KnownKumaScript):
+class XRefBase(KnownKumaScript):
+    """Base class for cross-reference KumaScript"""
+    expected_scopes = set((
+        'compatibility feature', 'specification description', 'footnote'))
+
+    def __init__(self, **kwargs):
+        super(XRefBase, self).__init__(**kwargs)
+        self.url = None
+        self.display = "(unset)"
+        self.linked = self.scope in ('specification description', 'footnote')
```

Comment:
Why are we determining `linked` behavior by `scope`? Should we instead determine it by the presence of a `url` value?